tests/integration/Core/Framework/ApiRoutesHaveASchemaTest.php

```diff
@@ -28,25 +28,26 @@ class ApiRoutesHaveASchemaTest extends TestCase
 
     private RouteCollection $routes;
 
-    private ContainerInterface $container;
-
     protected function setUp(): void
     {
-        $container = KernelLifecycleManager::bootKernel()->getContainer()->get('test.service_container');
-
-        static::assertInstanceOf(ContainerInterface::class, $container);
+        // Boot kernel, as some test definitions might still be registered in the old kernel
```

Comment:
Is it not possible for us to find which test adds them and fix that? We could also leave this here just in case, I'm just thinking it would be nice to wrap this cleaning up of definitions in a trait because it crops up quite a bit.

Reply:
Yes this is pretty much treating the symptom, I'll create a follow up ticket

Reply:
https://shopware.atlassian.net/browse/NEXT-40445